modules/monitoring-running-metrics-queries.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="running-metrics-queries_{context}"]
+= Running metrics queries
+
+You begin working with metrics by entering one or several Prometheus Query Language (PromQL) queries.
+
+.Procedure
+
+. Open the OpenShift Container Platform web console and navigate to *Monitoring* -> *Metrics*.
+
+. In the query field, enter your PromQL query.
+* To show all available metrics and PromQL functions, use the "Insert Metric at Cursor" button.
+. For multiple queries, use the "Add Query" button.
+. For deleting queries, click the action button for the query and select "Delete query".
```

Comment:
Is *12* earlier the _action button_? We had some discussion about how to describe the _..._ button.

We actually have an attribute for it:

`:kebab: image:kebab.png[title="Options menu"]`

So you could replace instances of these with:

> click {kebab} for the query, and then select **Delete query**.

s/"Delete query"/**Delete query**/

Reply:
Nice! I never liked my wording. Implemented for both occurrences.